src/main/java/org/folio/service/transformationfields/TransformationFieldsConfig.java

```diff
@@ -60,7 +64,9 @@
   PERMANENT_LOAN_TYPE_ID("permanentLoanTypeId", "$.item.permanentLoanTypeId", LOAN_TYPES),
   STATUS("status", "$.item.status.name"),
   VOLUME("volume", "$.item.volume"),
-  YEARCAPTION("yearCaption", "$.item.yearCaption");
+  YEARCAPTION("yearCaption", "$.item.yearCaption"),
+  ITEM_NOTE_TYPE("itemNoteTypeId", "$.item.notes[?(@.itemNoteTypeId=='{id}') && (!(@.staffOnly) || @.staffOnly == 'false'))].itemNoteTypeId", ITEM_NOTE_TYPES),
```

Comment:
```suggestion
  ITEM_NOTE_TYPE("itemNoteTypeId", "$.item.notes[?(@.itemNoteTypeId=='{id}' && (!(@.staffOnly) || @.staffOnly == 'false'))].itemNoteTypeId", ITEM_NOTE_TYPES),
```

Reply:
Please fix the paths, there is an extra ")" in all of them

Reply:
Good catch